.github/dependabot.yml

```diff
@@ -5,6 +5,10 @@ updates:
   schedule:
     interval: daily
     time: "03:00"
+  ignore:
+    - dependency-name: "dropwizard-dependencies"
+      versions:
+        - "> 3"
```

Comment:
I think you're better off doing ">= 4" otherwise it won't pick up minor versions of 3.

Also I've just realised I've done the Docker ignores wrong on some other repos 🙀

Reply:
I copied from another file!

Reply:
Updated.